builtin/providers/aws/resource_aws_elb_attachment.go

```diff
@@ -0,0 +1,120 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsElbAttachment() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsElbAttachmentCreate,
+		Read:   resourceAwsElbAttachmentRead,
+		Delete: resourceAwsElbAttachmentDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"elb": &schema.Schema{
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+
+			"instance": &schema.Schema{
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+		},
+	}
+}
+
+func resourceAwsElbAttachmentCreate(d *schema.ResourceData, meta interface{}) error {
+	elbconn := meta.(*AWSClient).elbconn
+	elbName := d.Get("elb").(string)
+
+	instance := d.Get("instance").(string)
+
+	registerInstancesOpts := elb.RegisterInstancesWithLoadBalancerInput{
+		LoadBalancerName: aws.String(elbName),
+		Instances:        []*elb.Instance{{InstanceId: aws.String(instance)}},
+	}
+
+	log.Printf("[INFO] registering instance %s with ELB %s", instance, elbName)
+
+	_, err := elbconn.RegisterInstancesWithLoadBalancer(&registerInstancesOpts)
+	if err != nil {
+		return fmt.Errorf("Failure registering instances with ELB: %s", err)
+	}
+
+	d.SetId(resource.PrefixedUniqueId(fmt.Sprintf("%s-", elbName)))
```

Comment:
I see what do you mean, but both fields are `ForceNew: true` anyway, so it's going to happen _anyway_.

My only concern about this is that it may make future resource importing slightly tricker. I haven't looked at the import functionality yet though, so maybe I'm wrong - in which case feel free to ignore this concern.